hazelcast/src/main/java/com/hazelcast/transaction/TransactionOptions.java

```diff
@@ -174,13 +174,10 @@ public void readData(ObjectDataInput in) throws IOException {
 
     @Override
     public String toString() {
-        final StringBuilder sb = new StringBuilder();
-        sb.append("TransactionOptions");
-        sb.append("{timeoutMillis=").append(timeoutMillis);
-        sb.append(", durability=").append(durability);
-        sb.append(", txType=").append(transactionType);
-        sb.append('}');
-        return sb.toString();
+        return "TransactionOptions"
+                + "{timeoutMillis=" + timeoutMillis
+                + ", durability=" + durability
+                + ", txType=" + transactionType + '}';
```

Comment:
Opening brace should be on first line in `"TransactionOptions"`.

Normally we have the ` + '}';` on a new line on multi-line messages.